src/action/darwin/create_fstab_entry.rs

```diff
@@ -193,8 +202,10 @@ fn fstab_entry(uuid: &Uuid, apfs_volume_label: &str) -> String {
 
 #[derive(thiserror::Error, Debug)]
 pub enum CreateFstabEntryError {
-    #[error("An `/etc/fstab` entry for the volume labelled `{0}` already exists. If a Nix Store already exists it may need to be deleted with `diskutil apfs deleteVolume \"{0}\") and should be removed from `/etc/fstab`")]
-    EntryExists(String),
+    #[error("An `/etc/fstab` entry for the `/nix` path already exists, consider removing the entry for `/nix` should be removed from `/etc/fstab`")]
+    NixEntryExists,
+    #[error("An `/etc/fstab` entry created by `nix-installer` already exists, if a volume named `{0}` already exists it may need to be deleted with `diskutil apfs deleteVolume \"{0}\" and the entry for `/nix` should be removed from `/etc/fstab`")]
```

Comment:
Nit:

```suggestion
    #[error("An `/etc/fstab` entry created by `nix-installer` already exists. If a volume named `{0}` already exists, it may need to be deleted with `diskutil apfs deleteVolume \"{0}\" and the entry for `/nix` should be removed from `/etc/fstab`")]
```